lib/queue.js

```diff
@@ -26,16 +28,9 @@ module.exports = FetchQueue;
 
 FetchQueue.prototype = [];
 FetchQueue.prototype.add = function(protocol, domain, port, path, depth, callback) {
-
-    // For legacy reasons
-    if (depth instanceof Function) {
-        callback = depth;
-        depth = 1;
-    }
-
     depth = depth || 1;
```

Comment:
If the block above is removed, legacy support will be broken (I think it should be OK, but just wanted a second opinion.)